recommendation/java/springboot/src/main/java/com/redhat/developer/demos/recommendation/RecommendationController.java

```diff
@@ -0,0 +1,71 @@
+package com.redhat.developer.demos.recommendation;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class RecommendationController {
+
+    private static final String RESPONSE_STRING_FORMAT = "recommendation v2 from '%s': %d\n";
```

Comment:
In Vertx implementation the default version is v1.

Reply:
I'll fix that and merge